config/initializers/assets.rb

```diff
@@ -0,0 +1,14 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Whats this file about?

Reply:
This file is for adding any additional assets to the asset pipeline outside of application.js, application.css, and all non-JS/CSS in the app/assets for compilation.